Documentation/concepts.rst

```diff
@@ -380,11 +380,24 @@ identities are prefixed with the string ``reserved:``.
 +---------------------+---------------------------------------------------+
 | Identity            | Description                                       |
 +=====================+===================================================+
-| reserved:host       | The host network namespace on which the pod or    |
-|                     | container is running.                             |
+| reserved:unknown    | The identity could not be derived.                |
++---------------------+---------------------------------------------------+
+| reserved:host       | The collection of all cluster hosts. Any traffic  |
+|                     | that originates from or is designated to one of   |
+|                     | the IPs of any host in the cluster is assigned the|
+|                     | reserved:host identity.                           |
 +---------------------+---------------------------------------------------+
 | reserved:world      | Any network endpoint outside of the cluster       |
 +---------------------+---------------------------------------------------+
+| reserved:health     | This is health checking traffic generated by      |
+|                     | Cilium agents.                                    |
++---------------------+---------------------------------------------------+
+| reserved:init       | An endpoint for which the identity has not be     |
```

Comment:
... for which the identity has not yet been resolved is assigned the init identity.